lib/grpc_rest.rb

```diff
@@ -105,16 +106,59 @@ def assign_params(request, param_hash, body_string, params)
       end
     end
 
+    def grpc_http_status(code)
```

Comment:
nit, might be helpful to link to grpc status codes https://grpc.github.io/grpc/core/md_doc_statuscodes.html

Also, might be good to put them in order